CHANGELOG.unreleased.md

```diff
@@ -22,6 +22,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 
 ### Fixed
 - Fixed a bug during dataset upload in case the configured `datastore.baseFolder` is an absolute path. [#8098](https://github.com/scalableminds/webknossos/pull/8098) [#8103](https://github.com/scalableminds/webknossos/pull/8103)
+- Fixed a bug where the materialize volume annotation job would not start due to a missing parameter. For now the name of the output segmentation layer of the job is autogenerated. [#8116](https://github.com/scalableminds/webknossos/pull/8116)
```

Comment:
I think this text matches #8118 instead ;)

Reply:
:woozy_face: Thanks for finding this :pray:

Reply:
Definitely too much pr switcheroo today :see_no_evil: